test/e2e/network/topology_hints.go

```diff
@@ -83,6 +83,7 @@ var _ = common.SIGDescribe("Feature:Topology Hints", func() {
 		framework.ExpectNoError(err, "timed out waiting for DaemonSets to be ready")
 
 		nodeNames := e2edaemonset.SchedulableNodes(c, ds)
+		e2eskipper.SkipIfNodesAllocatableCPUsNotEqual(c, nodeNames)
```

Comment:
I prefer to not add a new functions to skipper and resource until we have more evidence that is going to have more consumers, and doesn't seem something that can be reused, at least in a short term.
We can add the equivalent code of both functions to this test, just add comments explaining why they are needed

Reply:
done